services/web-app/utils/auth-checkers/validUserAuthorizationChecker.js

```diff
@@ -0,0 +1,22 @@
+import { retrieveUser } from '../retrieveUser'
+
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+/**
+ * Determines if user is authorized based on IBM email
+ * @param {GetServerSidePropsContext} context - getServerSideProps context object
+ * @returns {boolean} boolean indicating whether user is authorized to view resource or not
+ */
+const validUserAuthorizationChecker = async (context) => {
```

Comment:
Since this is looking specifically for ibm.com users and since it returns a boolean, I'd rename this function to something like `isValidIbmUser`

Reply:
Done!